### ISSUE TITLE
Crash when more then 4 Layer

### ISSUE THREAD

**Issue by SaraDark** [armory3d/armorpaint#633]
Crash when add more then 4 Layer (from materials or new layer)

Max eg:
![image](https://user-images.githubusercontent.com/69405892/96892690-2d1f4b00-148a-11eb-8ada-2420f05cb239.png)

When i add more crash.
ArmorPaint.org - v0.8 (2020-10-21) - 3e93019

**Comment by luboslenco** [comment]
Which OS are you on? I assume Linux? Which graphics api? (can be seen by clicking on `Menu bar - Help - About...`)

**Comment by luboslenco** [comment]
Oki I investigated a bit:
- This should only affect Linux, it's due to the texture limit in OpenGL.
- I pushed a fix at https://github.com/armory3d/armorpaint/commit/d95b89caf35ff12709045be82cbd7c6ce413b8f9, so it should not crash anymore.
- The problem remains that due to the texture limit for a single pass, not all layers can be displayed at once in the viewport. I am working to resolved this by switching to the Vulkan on Linux by default asap.

Many thanks for report!
